src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.AutoRedirect.cs

```diff
@@ -301,6 +329,13 @@ public async Task GetAsync_AllowAutoRedirectTrue_RedirectToUriWithParams_Request
         [InlineData(3, 4)]
         public async Task GetAsync_MaxAutomaticRedirectionsNServerHops_ThrowsIfTooMany(int maxHops, int hops)
         {
+            if (IsWinHttpHandler && !PlatformDetection.IsWindows10Version1703OrGreater)
+            {
+                // Skip this test if using WinHttpHandler but on a release prior to Windows 10 Creators Update.
+                _output.WriteLine("Skipping test due to Windows 10 version prior to Version 1703.");
```

Comment:
Does it show?IIRW output will be shown only in case of test fail, but maybe something changed.

Reply:
Again, it's not my code, I'm returning what there before [WinHttpHandler removal](https://github.com/dotnet/runtime/commit/ad8115d57708c491517b655bbdc0cbb619dda383#diff-843a3e2d5a9d5e9535c8deab8b47bcd7).

I can delete it, if you wish so, but it's not originally mine.

Reply:
> I can delete it, if you wish so, but it's not originally mine.

No no just curious, maybe things changed and I missed that, no worries.

Reply:
BTW, the ```_output.WriteLine``` is visible in testResults.xml even if the test succeeds. So it's handy at least for locally run tests where you can access all testResults.xml files.

Reply:
Yup that's true, I think it's here for that reason, usually used to enrich failure output.